### ISSUE TITLE
Message - Invalid handler specified

### ISSUE THREAD

**Issue by AnthonyJWinslow** [gazoodle/gecko-home-assistant#115]
## Version of the custom_component
HA Core: 2024.10.2
Ha Frontend: 20241002.3
Gecko: v0.1.11

## Configuration

```
No YAML, all through HACS & UI.
```

## Describe the bug

```
First time using the HACS gecko integration. While trying to setup through the UI, I receive the following message;

Error/n Config flow could not be loaded: {"message":"Invalid handler specified"}


Steps taken:
1. + Add Integration
2. Search for Gecko
3. Select Gecko
4. Error Message appears

```

## Debug log

```
Logger: homeassistant.config_entries
Source: config_entries.py:2945
First occurred: 2:48:04 PM (4 occurrences)
Last logged: 7:24:14 PM

Error occurred loading flow for integration gecko: No module named 'geckolib'
```

**Comment by AnthonyJWinslow** [comment]
Solved, folders (geckolib, geckolib-0.4.15.dist-info) are being installed to the wrong location. Moved them from:

**_Home Assistant\Configuration\deps_**

to

**_Home Assistant\Configuration\deps\lib\python3.12\site-packages_**

This solved the issue.

**Comment by AnthonyJWinslow** [comment]
Solved, folders (geckolib, geckolib-0.4.15.dist-info) are being installed to the wrong location. Moved them from:

**_Home Assistant\Configuration\deps_**

to

**_Home Assistant\Configuration\deps\lib\python3.12\site-packages_**

This solved the issue.
